requirements.txt

```diff
@@ -0,0 +1,4 @@
+meson
+ninja
+looseversion
+pytest
```

Comment:
Does this file support comments? If this stays in the project root, it needs to explain what it is for. Otherwise should be moved to .github/workflows.

Reply:
Yes it does.  requirements.txt is standard practice for anyone using python so I'd expect most people to recognise/understand it but as you say a comment at the top of the file would help.